app/requirements.txt

```diff
@@ -26,6 +26,7 @@ flask-restx>=0.2.0
 Flask-SQLAlchemy>=2.4,<3
 
 gitpython
+gumby @ git+https://github.com/WildMeOrg/gumby.git
```

Comment:
Does this need to be added to existing local repos like a submodule, or will a docker-compose.rebuild bring it in?